src/services/accounts/AccountsVestingInfoService.ts

```diff
@@ -28,7 +28,11 @@ export class AccountsVestingInfoService extends AbstractService {
 
 		return {
 			at,
-			vesting: vesting.isNone ? {} : vesting.unwrap(),
+			vesting: vesting.isNone
+				? []
+				: Array.isArray(vesting.unwrap())
+				? vesting.unwrap()
+				: [vesting.unwrap()],
```

Comment:
Would like to avoid double unwrap here, but not sure what a good way would be, TS would likely scream about something like:

```suggestion
				: Array.isArray(vesting = vesting.unwrap())
				? vesting
				: [vesting],
```

Could use a different variable for the unwrapped value (declared before the return, assigned in place)? :thinking:

Reply:
Ahh very good point. I should have noticed that earlier when I tried to abstract the `vesting.unwrap()` before calling vesting.isNone which caused me issues. 

Good call. Going to declare 2 variables, one for the bool value, and the other to unwrap vesting